platform/index.md

```diff
@@ -1,18 +1,70 @@
 - [Releases and Release Notes](release-notes.md)
 - [Platform Installer](installer)
 
-### Applications
+Everything started with a simple <cite>wrapper library around System Messaging API [1]</cite>. Few years later, it was shared with wider public in the form of an open-source project. Following the popular .NET convention it was called simply NServiceBus. The project gained popularity and soon it became clear that it would be difficult to keep meeting user demands without a dedicated team of developers. That is how the Particular Software company came to life. 
 
+Messaging is fantastic for building loosely coupled, scalable and reliable systems. However, it also has challenges. Two most common pains are having visibility of what is happening in the system and monitoring. In the beginning, teams using NServiceBus relied on generic monitoring applications or they built their own. After a while it became obvious that they could benefit from tools tailored to specific needs of NServiceBus. The other challenge is to understand the principles of building distributed systems and applying them in practice. This might be especially difficult for less experienced teams or developers that worked in a different way for many years. 
+
+That is how the idea of the Particular Service Platform was born, its goal is to provide a set of tools that 'just work' and make building and maintaining messaging systems easier, enabling the developers to focus on more important problems, such as understanding business domain rather than infrastructure. At the moment, Particular Service Platform consists of [NServiceBus](/nservicebus), [ServiceMatrix](/servicematrix), [ServiceControl](/servicecontrol), [ServiceInsight](/serviceinsight) and [ServicePulse](/servicepulse).
+
+<img src="architecture_overview.png" title="Particular Service Platform architecture">
+
+# NServiceBus - where it all begins
+
+NServiceBus is the heart of the system. It is a messaging and workflow framework that will help you to create distributed systems that are scalable, reliable and easy to modify. It supports various [messaging patterns](nservicebus/messaging/messages-events-commands.md), such as commands, events (publish-subscribe), offers support for handling long-running business processes in the form of [sagas](nservicebus/sagas/) and provides abstraction over multiple [queueing technologies](nservicebus/transports/), which guarantees that each message will be delivered once and only once. It automatically solves intermittent problems, by retrying messages and if that does not solve the issue, it sends them to the error queue, where they wait for human intervention.
+
+NServiceBus is very extensible, you can tailor it to your needs, select technologies you are already familiar with or create customized versions of various elements in the system. For more details refer to the [documentation](nservicebus/).
```

Comment:
> For more details refer to the [documentation](nservicebus/).

That is basically a dead end link since it is the generic index for nsb and in no way specific to the extensiblity points